docs/cli.md

```diff
@@ -97,6 +97,17 @@ pixi install --frozen
 pixi install --locked
 ```
 
+To reinitialize the lockfile in your project, you can remove the existing `pixi.lock` file and run `pixi install`.
+This process will regenerate the lockfile based on the dependencies defined in your manifest file:
+
+```sh
+rm pixi.lock && pixi install
+```
+This action ensures that your project's dependencies are reset and updated according to the current specifications in manifest file.
+
+In a future version of `pixi`, we will introduce the `pixi update` command.
```

Comment:
```suggestion
In a future version of `pixi`, we will introduce the `pixi update` command, see [#73](https://github.com/prefix-dev/pixi/issues/73).
```

Maybe also lead people to the place they need to watch to get notified of this feature?